docs/_includes/sum-section.adoc

```diff
@@ -34,4 +34,10 @@ Included in:
 |+:sectnums:+
 |Section numbering is off by default.
 Can be toggled on or off through document.
+
+|sectnumlevels
+|1-9
```

Comment:
The correct range is 0-5. (0 is effectively equivalent to disabling section numbering).